src/java/org/apache/cassandra/tools/nodetool/GcStats.java

```diff
@@ -19,19 +19,29 @@
 
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import org.apache.cassandra.tools.nodetool.stats.*;
 
 import io.airlift.airline.Command;
+import io.airlift.airline.Option;
 
 @Command(name = "gcstats", description = "Print GC Statistics")
 public class GcStats extends NodeToolCmd
 {
+    @Option(title = "format",
```

Comment:
Since now I have both `json` and `yaml` as options. I've kept the @Option annotation in the multiline format
`@Option(title = "format",
            name = {"-F", "--format"},
            description = "Output format (json, yaml)")
    private String outputFormat = "";`
Does this look good to you?